### PR TITLE
CacheClient - Fix misplaced root

### PR DESCRIPTION
**Pull request by GhenadieVP**
Jira: https://radixdlt.atlassian.net/browse/ABW-3388

## Description
Fixed misplaced folder root

https://github.com/radixdlt/babylon-wallet-ios/assets/118184705/ef370c9e-6428-4203-b7e1-80619e4810e4

### CODE REVIEW

File: RadixWallet/Clients/CacheClient/CacheClient+Interface.swift [hunk 1]
```diff
@@ -134,10 +134,10 @@ extension CacheClient {
 				case let .address(address):
 					return address.filesystemFilePath(folderPath: filesystemFolderPath)
 				case let .nonFungibleData(nonFungibleGlobalId):
-					return "\(filesystemFolderPath)/\(nonFungibleGlobalId.description)"
+					return "\(nonFungibleGlobalId.description)"
 				case let .nonFungibleIdPage(_, resourceAddress, pageCursor):
 					let file = "nonFungibleIds-" + resourceAddress.address + (pageCursor.map { "-\($0)" } ?? "")
-					return "\(filesystemFolderPath)/\(file)"
+					return "\(file)"
```
Comment (kugel3) [inline RadixWallet/Clients/CacheClient/CacheClient+Interface.swift:140]: i guess we could skip the string interpolation now that it's a single string

File: RadixWallet/Clients/CacheClient/CacheClient+Interface.swift [hunk 2]
```diff
@@ -200,6 +198,8 @@ extension CacheClient {
 			case .dateOfFirstTransaction:
 				"DateOfFirstTransaction"
```
Comment (kugel3) [inline RadixWallet/Clients/CacheClient/CacheClient+Interface.swift:199]: Maybe take the opportunity to turn this into a local constant?

Comment (CyonAlexRDX) [reply RadixWallet/Clients/CacheClient/CacheClient+Interface.swift:199]: hopefully translated into Sargon soon :)

**Review by CyonAlexRDX — APPROVED** [review]
lgtm

**Pull request merged**